Week1/MAKEME.md

```diff
@@ -7,18 +7,23 @@
 
 ### 1. Testing Exercises
 
+Using **Slack** application on desktop and mobile:
+
+- Describe shared testing activities for desktop and mobile versions
```

Comment:
Maybe you can be more specific with the **amount** of testing activities: like **at least 3 activities for desktop and 3 activities for mobile**.